src/bindings/python/src/openvino/frontend/pytorch/torchdynamo/backend.py

```diff
@@ -0,0 +1,67 @@
+import logging
+import os
+import torch
+from torch._dynamo.backends.common import fake_tensor_unsupported
+from torch._dynamo.backends.registry import register_backend
+from torch._inductor.compile_fx import compile_fx
+
+from openvino.runtime import Core, Type, PartialShape
+from openvino.frontend import FrontEndManager
+from openvino.frontend.pytorch.decoder import TorchScriptPythonDecoder
+
+log = logging.getLogger(__name__)
+
+
+@register_backend
+@fake_tensor_unsupported
+def openvino(subgraph, example_inputs):
+    return ts_openvino(subgraph, example_inputs)
+
+
+def ts_openvino(subgraph, example_inputs):
+    try:
+        model = torch.jit.script(subgraph)
+        model.eval()
+        fr_model = torch.jit.freeze(model)
+
+        core = Core()
+        fe_manager = FrontEndManager()
+        fe = fe_manager.load_by_framework('pytorch')
+        dtype_mapping = {
+            torch.float32: Type.f32,
+            torch.float16: Type.f16,
+            torch.int64: Type.i64,
+            torch.int32: Type.i32,
+            torch.uint8: Type.u8,
+            torch.int8: Type.i8,
+            torch.bool: Type.boolean
+        }
+        decoder = TorchScriptPythonDecoder(fr_model)
+
+        im = fe.load(decoder)
+        om = fe.convert(im)
+
+        for idx, input_data in enumerate(example_inputs):
+            om.inputs[idx].get_node().set_element_type(dtype_mapping[input_data.dtype])
+            om.inputs[idx].get_node().set_partial_shape(PartialShape(list(input_data.shape)))
+        om.validate_nodes_and_infer_types()
+
+        device = 'CPU'
+        if (os.getenv("OPENVINO_DEVICE") is not None):
```

Comment:
Can we consider more "specialized" name for that? I mean `OPENVINO_DEVICE` looks too global that can be mixed with other part of APIs, like default device in `compile_model`. Not a big deal, but if you have other options like `OPENVINO_TORCH_BACKEND_DEVICE` it is worth considering.

Another question is can we really avoid using such env variables and provide the device as a parameter of torch.compile with our backend? I don't know about technical possibility and not sure about convenience in all the scenarios, but it is more "local" solution that doesn't allow any wrong associations.

Reply:
As of now, we don't have the option to add any config parameters to torch.compile. It's only available for inductor backend

Reply:
But we can use your suggestion for changing the device name